### PR TITLE
Add Wormhole Gateway chain and initial assetlist

### PR DESCRIPTION
**Pull request by nik-suri**

### CODE REVIEW

File: gateway/chain.json [hunk 1]
```diff
@@ -0,0 +1,128 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "gateway",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://wormhole.com",
+  "pretty_name": "Wormhole Gateway",
+  "chain_id": "wormchain",
+  "bech32_prefix": "wormhole",
```
Comment (JeremyParish69) [inline gateway/chain.json:9]: Please register this prefix: https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Comment (nik-suri) [reply gateway/chain.json:9]: @JeremyParish69 see https://github.com/satoshilabs/slips/pull/1590

File: gateway/assetlist.json [hunk 2]
```diff
@@ -0,0 +1,411 @@
...
+      "description": "The native fee token of Gateway.",
+      "denom_units": [
+        {
+          "denom": "utest",
+          "exponent": 0,
+          "aliases": []
+        },
+        {
+          "denom": "test",
+          "exponent": 0,
```
Comment (nik-suri) [inline gateway/assetlist.json:15]: This exponent should be `6`

File: gateway/assetlist.json [hunk 3]
```diff
@@ -0,0 +1,411 @@
...
+      "description": "the native staking token of Gateway.",
+      "denom_units": [
+        {
+          "denom": "uworm",
+          "exponent": 0,
+          "aliases": []
+        },
+        {
+          "denom": "worm",
+          "exponent": 0,
```
Comment (nik-suri) [inline gateway/assetlist.json:35]: This exponent should be `6`

**Pull request merged**